cognite/powerops/client/shop_result_files.py

```diff
@@ -121,6 +120,7 @@ def find_time_series(
         matches_attribute_name="",
     ) -> list[str]:
         """Find time series in the results file. Some keys are parsed as numbers"""
+        # TODO: accept either a single string or a list of strings
```

Comment:
the idea being that if several keys are supplied, we could plot them as subfigures. However, this might not be needed since they can always just generate a new plot with a the new key